src/__tests__/components/Notification/Notification.test.js

```diff
@@ -2,37 +2,40 @@ import React from 'react';
 import Enzyme, { mount } from 'enzyme';
 import EnzymeAdapter from 'enzyme-adapter-react-16';
 import { Provider } from 'react-redux';
-import { MemoryRouter } from 'react-router-dom';
-import { mockStore, initialState } from '../../../__mocks__/store';
+import thunk from 'redux-thunk';
+import configureMockStore from 'redux-mock-store';
 import notification from '../../../__mocks__/notification';
 import Notification from '../../../components/Profile/Settings/NotificationsComponent/Notification';
 
 Enzyme.configure({ adapter: new EnzymeAdapter() });
```

Comment:
@MBenedicte, it would be better if you use the configuration file where we have enzyme configured.